website/src/contribute/index.md

```diff
@@ -149,9 +149,11 @@ To contribute code, you need
    have an open source license [compatible](https://www.apache.org/legal/resolved.html#criteria) with Apache.
 1. Add unit tests for your change
 1. When your change is ready to be reviewed and merged, create a pull request.
-   Format the pull request title like `[BEAM-XXX] Fixes bug in ApproximateQuantiles`,
+   Format commit messages and the pull request title like `[BEAM-XXX] Fixes bug in ApproximateQuantiles`,
```

Comment:
I think some time ago we had a line that was something like "for trivial changes this is not necessary", when the rule applied only to pull requests. Is that line deleted? Just checking. I am actually fine having it deleted. I think that someone should be very involved in the project enough to have appropriate judgment about when it might be overkill.

Reply:
Not aware of the history of that line. But in the spirit of keeping instructions concise, we should probably leave the expert mode out from here?